docs/how-to-guides/deploy-a-flow/deploy-using-docker.md

```diff
@@ -83,7 +83,38 @@ name: open_ai_connection
 module: promptflow.connections
 api_key: ${env:OPEN_AI_CONNECTION_API_KEY} # env reference
 ```
-You'll need to set up the environment variables in the container to make the connections work.
+You'll need to set up the environment variables in the container to make the connections work. For a full list of environment variables see [Load from environment variables](https://microsoft.github.io/promptflow/how-to-guides/manage-connections.html#load-from-environment-variables).
```

Comment:
same as the comment in another doc,  this "manage-connections" link is not for deployment, we need a section for this in the deployment doc

Reply:
Same with this one. Would it be better to include a section for configuring the connection within this document rather than linking to the `manage-connections` page?

Reply:
I've added a section with example to the same docker how to doc in [931705c](https://github.com/microsoft/promptflow/pull/3508/commits/931705c12f33d8dbea260e22679c68d81aa09893).